radio/src/translations.cpp

```diff
@@ -236,6 +236,10 @@ const char STR_MENULOGICALSWITCH[] = TR_MENULOGICALSWITCH;
 const char STR_MENULOGICALSWITCHES[] = TR_MENULOGICALSWITCHES;
 const char STR_MENUCUSTOMFUNC[] = TR_MENUCUSTOMFUNC;
 const char STR_SPLASHSCREEN[] = TR_SPLASHSCREEN;
+#if defined(PWR_BUTTON_PRESS)
```

Comment:
I am not sure it's needed to add this `#if defined`, the optimizer should do his job